signbank/dictionary/templates/dictionary/gloss_detail_advanced.html

```diff
@@ -75,24 +75,14 @@ <h2 id='modalTitle'>{% blocktrans %}Delete This Morphology{% endblocktrans %}</h
             </table>
 
             {% if perms.dictionary.change_gloss %}
-            <form id='add_morphologydefinition_form' method='post' action='{% url 'dictionary:add_morphologydefinition' %}'>
```

Comment:
I think we need to be careful with this. The way FinSL and I think GlobalSignbank is using Morphology is to relate signs to one another - this is why there's a "Add Morphology Definition" form here. Can we keep this form here for now and _also_ show the morphology table?

Reply:
@joshmcarthur , I have added the form back, if it looks all good, I'll do the same for other opening PRs as well :slightly_smiling_face: